DebbeqGasha/DG001.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DGDG001" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Sane Golgotā</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="GA"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0965DGDG"/>
+                        <idno>DGDG-001</idno>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        <summary/>
+                        
+                          <msItem xml:id="ms_i1">
+                              <locus from="4r" to="50v"/>
+                              <title type="complete" ref="LIT2246salotz"/>
+                          </msItem>
+                        
+                                <msItem xml:id="ms_i1.1">
+                                    <locus from="4r" to="4v"/>
```

Comment:
I think we are nearly finished :) I just have some questions about the division into subsections. You encode 4r-50v as LIT2246salotz, and then subdivide this into 4r-4v (with incipit) and 4v-50v (with explicit). Is there a new section beginning on 4v?

Reply:
it is an error there is no new section that begins in 4v. the new section is started at 51r. let me correct it.